### PR TITLE
adds order by in the getSiteList function

### PR DESCRIPTION
**Pull request by mohadesz**
Modified the getSiteList function by adding the "order by " in the query.

### CODE REVIEW

**Comment by ridz1208** [comment]
@mohadesz run phpcs

File: php/libraries/Utility.class.inc [hunk 1]
```diff
@@ -188,7 +188,8 @@ class Utility
         if ($study_site) {
             $query .= "WHERE Study_site='Y'";
         }
-
+        $order_by = " ORDER BY Name";
```
Comment (alisterdev) [inline php/libraries/Utility.class.inc]: The variable seems unnecessary `$query .= " ORDER BY Name";` is clear enough

**Comment by ridz1208** [comment]
@mohadesz This PR does not do what it is supposed to. Since it is returning an associative array, the ordering in the query does not really matter, its the order of the CenterID that is respected on the front end.

Do you want to fix this ? or log a redmine ticket and close this ?

**Comment by mohadesz** [comment]
@ridz1208  I Created a redmine ticket for this https://redmine.cbrain.mcgill.ca/issues/12092 so I am closing it.

**Pull request closed without merging**